docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -48,19 +48,46 @@ This module includes the internal `assert_only_owner` to restrict a function to
 * xref:OwnableComponent-renounceOwnership[`++renounceOwnership(self)++`]
 --
 
+[.contract-index]
+[[OwnableTwoStepImpl]]
+.Embeddable Implementations (two step transfer)
```

Comment:
```suggestion
.Embeddable Implementations (Two Step Transfer)
```
this is written differently throughout the page: "two step transfer", "Two Step Transfer", and "Two step transfer". We should stick with one, probably "Two Step Transfer" which is consistent with the Title Casing, although "Two step transfer" looks better

Reply:
Used "Two Step Transfer" everywhere.